ports/espressif/boards/circuitart_zero_s3/board.h

```diff
@@ -0,0 +1,109 @@
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#ifndef CIRCUITART_ZERO_S3_H_
+#define CIRCUITART_ZERO_S3_H_
+
+//--------------------------------------------------------------------+
+// Button
+//--------------------------------------------------------------------+
+
+// Enter UF2 mode if GPIO is pressed while 2nd stage bootloader indicator
+// is on e.g RGB = Purple. If it is GPIO0, user should not hold this while
+// reset since that will instead run the 1st stage ROM bootloader
+#define PIN_BUTTON_UF2        0
+
+// GPIO that implement 1-bit memory with RC components which hold the
+// pin value long enough for double reset detection.
+#define PIN_DOUBLE_RESET_RC   45
+
+//--------------------------------------------------------------------+
+// LED
+//--------------------------------------------------------------------+
+
+// GPIO connected to Neopixel data
+#define NEOPIXEL_PIN          3
+
+// Brightness percentage from 1 to 255
+#define NEOPIXEL_BRIGHTNESS   0x10
+
+// Number of neopixels
+#define NEOPIXEL_NUMBER       1
+
+// LED for indicator and writing flash
+// If not defined neopixel will be use for flash writing instead
+//#define LED_PIN               46
+//#define LED_STATE_ON          1
+
+//--------------------------------------------------------------------+
+// TFT
+//--------------------------------------------------------------------+
+
+#define CONFIG_LCD_TYPE_ST7789V
+
+#define DISPLAY_PIN_MISO      -1 // required if use CONFIG_LCD_TYPE_AUTO
+#define DISPLAY_PIN_MOSI      35
+#define DISPLAY_PIN_SCK       36
+
+#define DISPLAY_PIN_CS        39
+#define DISPLAY_PIN_DC        38
+#define DISPLAY_PIN_RST       4
+
+#define DISPLAY_PIN_BL        46
+#define DISPLAY_BL_ON          0  // GPIO state to enable back light
+
+//#define DISPLAY_PIN_POWER      -1
+//#define DISPLAY_POWER_ON       0  // GPIO state to enable TFT
+
+#define DISPLAY_WIDTH         320
+#define DISPLAY_HEIGHT        172
+
+#define DISPLAY_COL_OFFSET    53
+#define DISPLAY_ROW_OFFSET    40
+
+// Memory Data Access Control & // Vertical Scroll Start Address
+#define DISPLAY_MADCTL        (TFT_MADCTL_MX)
+#define DISPLAY_VSCSAD        0
+
+#define DISPLAY_TITLE         "Zero S3 TFT"
+
+//--------------------------------------------------------------------+
+// USB UF2
+//--------------------------------------------------------------------+
+
+#define USB_VID                  0x303A
+#define USB_PID                  0x80DC
+
+#define USB_MANUFACTURER         "CircuiArt"
+#define USB_PRODUCT              "ZeroS3"
+
+#define UF2_PRODUCT_NAME         USB_MANUFACTURER " " USB_PRODUCT
+#define UF2_BOARD_ID             "ESP32S3-Zero-R1"
+#define UF2_VOLUME_LABEL         "ZEROS3BOOT"
+#define UF2_INDEX_URL            "https://github.com/CircuitART"
+
+// Use favicon
+#define TINYUF2_FAVICON_HEADER   "favicon_adafruit_256.h"
```

Comment:
you probably forgot to remove this line. The favicon will show up Adafruit logo when using with windows.

Reply:
hi, oh okay got it will remove the FAVICON, but i also did some other changes as well which I will edit later.

by the way, as you might have noticed, my board has a built-in TFT (with no Pin for BL control) and i was wondering what I can type instead of the pin number for this
 #define DISPLAY_PIN_BL        46
i have tried -1 but it does not work because MISO has -1
i also used  NULL/null but i get errors during the build process

thanks

Reply:
oh, you will need to slightly update the code for the lcd component to opt out when DISPLAY_PIN_BL = -1 https://github.com/adafruit/tinyuf2/blob/master/ports/espressif/components/lcd/lcd.c

Reply:
i had a look at the lcd.c and i think its not within my knowledge so, for now, I used it as is.
thanks